vision/api/VisionTest/VisionTest.cs

```diff
@@ -72,6 +74,43 @@ public static byte[] ToPngBytes(string imagePath)
         }
     }
 
+    /// <summary>
+    /// Runs tests detecting text from PDF or TIFF files
+    /// These require uploading a .pdf to Google Cloud Storage
+    /// and then API results are saved to Google Cloud Storage
+    /// </summary>
+    public class PdfDocumentTests : LocalTests, IClassFixture<RandomBucketFixture>
+    {
+        readonly string _bucketName;
+        readonly BucketCollector _bucketCollector;
+        readonly string _pdfFileName = "HodgeConj.pdf";
+
+        public PdfDocumentTests(RandomBucketFixture bucketFixture)
+        {
+            _bucketName = bucketFixture.BucketName;
+            _bucketCollector = new BucketCollector(_bucketName);
+            var localPath = Path.Combine("data", _pdfFileName);
+            _bucketCollector.CopyToBucket(localPath, _pdfFileName);
+        }
+
+        [Fact(Skip = "Test hangs")]

```

Comment:
... this test gets run!? 🤯

----

<img width="406" alt="screen shot 2018-07-23 at 2 56 49 pm" src="https://user-images.githubusercontent.com/280/43105290-38de35fc-8e89-11e8-9528-abbf08f4ee60.png">

----

w.t.f. mate

Reply:
I'm mystified 0_o

Reply:
I isolated the cause of the hang: it's line 93. 

I'm going to use a similar technique as is used in the SpeechText project: defining an Upload() method in this class that uploads the local file to the GCS bucket.